templates/content/blank/NewApp.Android/NewApp.Android.fsproj

```
@@ -187,7 +187,7 @@
       <HintPath>..\packages/Newtonsoft.Json.NewtonsoftJsonPkg/lib/netstandard2.0/Newtonsoft.Json.dll</HintPath>
     </Reference>
     <Reference Include="Xamarin.Essentials">
-      <HintPath>..\packages/Xamarin.Essentials.0.6.0-preview/lib/monoandroid71/Xamarin.Essentials.dll</HintPath>
+      <HintPath>..\packages/Xamarin.Essentials.1.1.0/lib/monoandroid71/Xamarin.Essentials.dll</HintPath>
```

Oh, good catch. That's one mistake that has been missed for a long time apparently. 😄 
The version should only be managed by `template.json`. Here we should only have the keyword `XamarinEssentialsSdk` (will be replaced when created by `dotnet new`)

```suggestion
      <HintPath>..\packages/Xamarin.Essentials.XamarinEssentialsSdk/lib/monoandroid71/Xamarin.Essentials.dll</HintPath>
```

---

Right, better solution!